schema/wasi-args.capnp

```diff
@@ -0,0 +1,21 @@
+@0xb47a0b67171659f7;
```

Comment:
what are these?

Reply:
that is the unique file id: https://capnproto.org/language.html#unique-ids
the `@0`, `@1`, `@2`, etc. are used to define the canonical field/method order for seriaization.

Reply:
we could possible use these in the future for versioning the interfaces but I wanted to start on discussion at time.